src/integrators/verner_rk_integrators.jl

```diff
@@ -153,6 +161,12 @@ end
   integrator.kshortsize = 10
   k = eltype(integrator.sol.k)(integrator.kshortsize)
   integrator.k = k
+
+  # Avoid undefined entries if k is an array of arrays
+  prototype = f(integrator.t, integrator.uprev)
```

Comment:
what about just doing `uprev./t` then? That's always going to work and has essentially no cost.